### PR TITLE
Add an OEP-2 compliant openedx.yaml file

### PR DESCRIPTION
**Pull request by cpennington**
This adds an `openedx.yaml` file, as described by OEP-2:
http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html

The data in this file was transformed from the contents of
edx/repo-tools-data:repos.yaml

### CODE REVIEW

**Comment by cpennington** [comment]
@edx/edx-accessibility: Just a heads up, this PR is marking you as an owner of this repo.

File: openedx.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: a11y
+oeps: {}
+openedx-release: {ref: master}
```
Comment (benpatterson) [inline openedx.yaml]: what does this mean?

File: openedx.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+owner: edx/edx-accessibility
+track-pulls: true
```
Comment (benpatterson) [inline openedx.yaml]: and what does this mean?

File: openedx.yaml (hunk 1 above)
Comment (cpennington) [inline openedx.yaml]: This is used by the opensource team for automation when tagging named releases.

File: openedx.yaml (hunk 2 above)
Comment (cpennington) [inline openedx.yaml]: This specifies whether the repo should show up in various PR reports that the opensource team does.

**Comment by cpennington** [comment]
@benpatterson @edx/edx-accessibility: Is this ok to merge, at this point?

**Comment by benpatterson** [comment]
@cpennington yep!

👍

**Pull request merged**